test-support/helpers/ember-tooltips.js

```diff
@@ -112,6 +116,18 @@ function getTooltipAndTargetPosition(options = {}) {
   };
 }
 
+/* TODO(Duncan): Document */
+
+export function findTooltip(selector = tooltipOrPopoverSelector) {
```

Comment:
Technically this makes `findTooltip` and `findTooltipTarget` new public test helpers but I'm not going to document and officially release them until the PR for expanding and improving the test helpers documentation is released.